app/models/pglogical_subscription.rb

```diff
@@ -1,50 +1,56 @@
 require 'pg/dsn_parser'
 require 'pg/logical_replication'
+require 'active_hash'
 
-class PglogicalSubscription < ActsAsArModel
-  set_columns_hash(
-    :id                   => :string,
-    :status               => :string,
-    :dbname               => :string,
-    :host                 => :string,
-    :user                 => :string,
-    :password             => :string,
-    :port                 => :integer,
-    :provider_region      => :integer,
-    :provider_region_name => :string
-  )
```

Comment:
ActsAsArModel is supported by Reporting, and the data types here are what help drive the display of the "columns".  Does ActiveHash support the methods needed by Reporting?

Reply:
Not sure. Can you show me where Reporting is implemented and where it uses those datatypes?